packages/modules/web_themes/koala/source/src/components/ChargePointVehicleSelect.vue

```diff
@@ -49,7 +49,17 @@ const connectedVehicle = mqttStore.chargePointConnectedVehicleInfo(
   props.chargePointId,
 );
 
-const vehicles = computed(() => mqttStore.vehicleList);
+const hideStandardVehicle = computed(
+  () => mqttStore.themeConfiguration?.hide_standard_vehicle,
+);
+const vehicles = computed(() => {
+  if (hideStandardVehicle.value) {
+    return mqttStore.vehicleList.filter(
+      (vehicle) => vehicle.name !== 'Standard-Fahrzeug',
+    );
+  }
+  return mqttStore.vehicleList;
+});
```

Comment:
Du hast jetzt in `ChargePointVehicleSelect` und `VehicleInformation` die selbe Funktionalität umgesetzt. Es wäre besser, wenn der Code nur einmalig vorhanden ist. Es bietet sich daher an, das Standard-Fahrzeug bereits im MQTT-Store in `vehicleList()` zu entfernen.